src/base/Text.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Text, TextProps } from "react-native";
+import { useTheme } from "../context/ThemeContext";
+
+interface TextComponentProps extends TextProps {
+	className?: string;
+	content: string | number | undefined;
+	bold?: boolean;
+}
+
+const TextComponent = ({ bold, className = "", style, content, ...props }: TextComponentProps) => {
+	const { themeVariables } = useTheme();
+
+	const finalClassName = `${className} ${bold ? "font-bold" : "font-normal"}`;
+
+	return (
+		<Text className={finalClassName} style={[{ color: themeVariables.text }, style]} {...props}>
```

Comment:
Rajouter une props `isError?: boolean` et du coup l'utiliser comme ceci : 
```suggestion
		<Text className={finalClassName} style={[{ isError ? color: themeVariables.textError : color: themeVariables.text }, style]} {...props}>
```

Comme ça pour la page de login tu passes la props isError à true et le text sera rouge